mkosi/__init__.py

```diff
@@ -5001,6 +5006,20 @@ def _format_action_invocation(self, action: argparse.Action) -> str:
         args_string = self._format_args(action, default)
         return ", ".join(action.option_strings) + " " + args_string
 
+    def _split_lines(self, text: str, width: int) -> List[str]:
```

Comment:
Am I missing something or is this function unused?

Reply:
I just wanted to ask how that was picked up, but then I finally saw the `self` and realised it wasn't a freestanding function, but a method. Nevermind then.